code/game/objects/items/handcuffs.dm

```diff
@@ -60,7 +60,8 @@
 	if(!C.handcuffed)
 		if(C.canBeHandcuffed())
 			C.visible_message(span_danger("[user] is trying to put [src.name] on [C]!"), \
-								span_userdanger("[user] is trying to put [src.name] on you!"))
+								span_userdanger("[user] is trying to put [src.name] on you!"), \
+								span_userdanger("You feel someone grab your wrists, the clink of [src.name] rattling around your ears!"))
```

Comment:
```suggestion
								span_userdanger("[user] is trying to put [name] on you!"), \
								span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"))
```
There's no need to specify that atom of the var access operation is `src`, it's already implied.